# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1466779024 14400
#      Fri Jun 24 10:37:04 2016 -0400
# Node ID 3c3995706dbe597f871be01f79bf8d423675ba11
# Parent  89423997486819a9705d92f123e6f005d4d6fa37
* find-files-with-tests.sh: Add missing copyright info.

diff --git a/build-aux/find-files-with-tests.sh b/build-aux/find-files-with-tests.sh
--- a/build-aux/find-files-with-tests.sh
+++ b/build-aux/find-files-with-tests.sh
@@ -1,9 +1,27 @@
 #! /bin/sh
+#
+# Copyright (C) 2013-2016 John W. Eaton
+#
+# This file is part of Octave.
+#
+# Octave is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at
+# your option) any later version.
+#
+# Octave is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Octave; see the file COPYING.  If not, see
+# <http://www.gnu.org/licenses/>.
 
 set -e
 
 GREP=${GREP:-grep}
 SED=${SED:-sed}
 
 srcdir="$1"
 if [ "$1" ]; then
